aws/resource_aws_elastic_transcoder_pipeline_test.go

```diff
@@ -279,10 +266,11 @@ EOF
 }
 
 resource "aws_s3_bucket" "test_bucket" {
-  bucket = "aws-elasticencoder-pipeline-tf-test-bucket-%[1]s"
+  bucket = %q
   acl    = "private"
 }
-`
+`, rName, rName, rName)
```

Comment:
You can use argument indexes, shown in my comments above, to avoid passing in the same argument multiple times.

Reply:
Updated 👍